bloom/commands/git/patch/trim_cmd.py

```diff
@@ -104,7 +104,7 @@ def _trim(config, force, directory):
             items.append(item)
         # Remove and .* files missed by 'git rm -rf *'
         if len(items) > 0:
-            execute_command('git rm -rf ' + ' '.join(items), cwd=directory)
+            execute_command('git rm -rf ' + ' '.join(['"%s"' % i for i in items if i]), cwd=directory)
```

Comment:
I don't know why I inverted the quotes here relative to the others. I'm flipping this around so that single quotes are used in the result string to avoid any shell expansion.